compositor_render/src/renderer/node.rs

```diff
@@ -1,12 +1,17 @@
 use compositor_common::renderer_spec::RendererId;
 
+use crate::transformations::shader::validation::ParametersValidationError;
+
 // TODO: move here Node and RenderNode types
 
 #[derive(Debug, thiserror::Error)]
 pub enum CreateNodeError {
     #[error("Shader \"{0}\" does not exist. You have to register it first before using it in the scene definition.")]
     ShaderNotFound(RendererId),
 
+    #[error("Error while validating the parameters for the shader node:\n{0}")]
```

Comment:
My intention was that we always do sth like
```
Error while validating the parameters for the shader node. {0}
```
and `:\n{0}` is only used when under {0} we have sth multiline that is not error and describes. If we are not careful we will land with more cases like this:

```
    The vertex shader input has a wrong type. Structure mismatch found while checking the type of field texture_id in struct VertexInput:
    Type internal structure doesn't match:
        Expected:
    Scalar {
        kind: Sint,
        width: 4,
    }

    Actual:
    Scalar {
        kind: Uint,
        width: 4,
    }.
```
were we have 2 lines ending with `:`

Reply:
I don't get it, are two lines that end with `:` that bad?